component-sets/material/ui/styling/material_style_metrics.slint

```diff
@@ -8,10 +8,12 @@ export global MaterialStyleMetrics {
     out property <length> size_4: 4px;
     out property <length> size_6: 6px;
     out property <length> size_16: 16px;
+    out property <length> size_18: 18px;
     out property <length> size_30: 30px;
     out property <length> size_32: 32px;
     out property <length> size_36: 36px;
     out property <length> size_40: 40px;
+    out property <length> size_48: 48px;
```

Comment:
Not related to this PR, but IMO we should change the material style to match this or make this match the material style. (rem is the solution that works when people for example increase the system font size on the mobile phones because of their degraded vision)